Dockerfile_intel

```diff
@@ -36,7 +38,8 @@ RUN cargo build --profile release-opt
 
 
 # Text Generation Inference base image for Intel
-FROM intel/intel-extension-for-pytorch:2.1.30-xpu as base
+
+FROM intel/intel-extension-for-pytorch:2.1.30-xpu as xpu
```

Comment:
Should we update PyTorch to a more recent version? Or do you prefer to stick to this one?

Reply:
for xpu. we stick to this one currently, We plan to release new version in end of July. and I will update the dockerfile at that time.